src/Tes.Runner/Events/EventsPublisher.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Storage.Sas;
+using Microsoft.Extensions.Logging;
+using Tes.Runner.Models;
+using Tes.Runner.Storage;
+using Tes.Runner.Transfer;
+
+namespace Tes.Runner.Events;
+
+public class EventsPublisher : IAsyncDisposable
+{
+    const string EventVersion = "1.0";
+    const string EventDataVersion = "1.0";
+    public const string TesTaskRunnerEntityType = "TesRunnerTask";
+    public const string DownloadStartEvent = "downloadStart";
```

Comment:
Should we add all of these to a Wiki page docs?

Reply:
Yes, I agree.